### ISSUE TITLE
Error with separator and negative numbers

### ISSUE THREAD

**Issue by brammitch** [inorganik/countUp.js#190]
When the minus sign appears in front of a 3 or 6 digit number, the separator is placed behind the minus sign, as though the minus sign were in the thousands or millions place value.

Here are four examples of how this has shown up in our app:

- Correct behaviors
```
// Result: -95
// Expected: -95

// Result: -22,688
// Expected: -22,688
```

- Incorrect behaviors
```
// Result: -,111
// Expected: -111

// Result: -,178,238
// Expected: -178,238
```

This is easily reproducible by entering "-111" or "-178238" on the [demo page](http://inorganik.github.io/countUp.js/).

Our current workaround is to check the the value is negative, then multiply by -1 and add a prefix if it is.

**Comment by inorganik** [comment]
Yikes! Thanks for the heads up, on it

**Comment by inorganik** [comment]
Fixed now in [1.9.3](https://github.com/inorganik/countUp.js/releases/tag/v1.9.3).

**Comment by brammitch** [comment]
Great! Thanks @inorganik :+1:
